### ISSUE TITLE
"Unable to determine time zone" should be logged, not printed

### ISSUE THREAD

**Issue by lefth** [ogham/exa#980]
System:
- v0.10.1
- command: `ls -l`
- operating system: linux, OpenWrt, a recent snapshot. An arm7 musl system.

What happens:
"Unable to determine time zone: No such file or directory (os error 2)" is printed to stdout.

What should happen:

- This string should be logged, not shown--because time zones are an optional feature. The tool should still work with UTC.
- If the string is printed, it should go to stderr, so it won't become part of output redirection. After all, the error message is not a file. It shouldn't be considered part of the data displayed by exa.

**Comment by ariasuni** [comment]
I’m wondering, could you try the changes in this PR and tell if they fix your issue?
https://github.com/ogham/exa/pull/867

**Comment by lefth** [comment]
No, I had to install time zone info and reboot. But thank you. And my point in filling this bug is that I don't expect to see error messages in every result on systems that don't have time zone into installed. `ls` doesn't do that, and other programs don't do that.
